src/server.ts

```diff
@@ -78,41 +78,46 @@ const startServer = async () => {
 
   // Unique name/email/organisation check
   server.get('/check-unique', async (request: any, reply) => {
-    const { type, value } = request.query
+    const { type, value, table, field } = request.query
     if (value === '' || value === undefined) {
       reply.send({
         unique: false,
         message: 'Value not provided',
       })
       return
     }
-    let table, field
+    let tableName, fieldName
     switch (type) {
       case 'username':
-        table = 'user'
-        field = 'username'
+        tableName = 'user'
+        fieldName = 'username'
         break
       case 'email':
-        table = 'user'
-        field = 'email'
+        tableName = 'user'
+        fieldName = 'email'
         break
       case 'organisation':
-        table = 'organisation'
-        field = 'name'
+        tableName = 'organisation'
+        fieldName = 'name'
         break
       case 'orgRegistration':
-        table = 'organisation'
-        field = 'registration'
+        tableName = 'organisation'
+        fieldName = 'registration'
```

Comment:
Hmm I'm wondering what is this field now? WOuld this be also a generated one?

Reply:
Yeah, good point, this isn't meaningful anymore, I'll remove it -- b56ed1e5